pegasus/sites.v3/code.org/public/css/music-lab-hero-banner.scss

```diff
@@ -0,0 +1,118 @@
+$width-md: 1145px;
+$width-sm: 640px;
+
+#bars {
+  max-width: 1240px;
+  height: 480px;
+  position: relative;
+  overflow: hidden;
+  align-items: center;
+  justify-content: center;
+
+  @media (max-width: $width-sm) {
+    height: 550px;
+  }
+
+  & > div.col-33 {
+    position: absolute;
+    background-position: center !important;
+
+    // Copy and buttons
+    &:nth-of-type(2) {
+      width: 560px;
+      top: unset;
+      left: unset;
+      transform: unset;
+      padding-bottom: 0 !important;
```

Comment:
Using an `!important` here since this needs to override styles that are set inline on this element.